src/java.base/share/classes/java/io/InputStream.java

```diff
@@ -594,20 +594,19 @@ public long skip(long n) throws IOException {
      * @see        java.io.InputStream#skip(long)
      */
     public void skipNBytes(long n) throws IOException {
```

Comment:
Not related to this change, but looks like `@since` is missing.

Reply:
https://bugs.openjdk.java.net/browse/JDK-8256183